.github/workflows/diff_report.yml

```diff
@@ -114,29 +127,22 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: ${{ env.AWS_REGION }}    
          
-     - name: Generate basic difference report
-       if: github.event.comment.body == 'diff report'
+     - name: Generate report
        run: |
          cd contribution/checkstyle-tester
          bash
          REF="origin/${{needs.parse_body.outputs.branch}}"
          REPO="../../checkstyle"
-         if [ -f patch_config.xml ]; then
-          groovy diff.groovy -r $REPO -b master -p $REF -bc config.xml -pc patch_config.xml\
+         if [ -f new_module_config.xml ]; then
+          groovy diff.groovy -r $REPO -p $REF -pc new_module_config.xml -m single\
+          -l project.properties
```

Comment:
+2 spaces for indentation. The same for other linewraps.

Reply:
done in separate commit